### DISCUSSION TITLE
Resources feedback

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#835] [Feature feedback]
Let's use this space to provide feedback for anything resource-related.

Post [an issue](https://github.com/avo-hq/avo/issues/new/choose) if something is broken.

**Comment by brandonzylstra** [comment]
https://docs.avohq.io/2.0/resources.html#defining-resources is linked to as a next step after installing Avo, but nowhere is there any indication of what URL to go to to see your new resources.

**Reply by adrianthedev** [reply]
Fixed it [here](https://docs.avohq.io/2.0/installation.html#:~:text=This%20will%20mount%20the%20app%20under%20/avo%20path.%20Visit%20that%20link%20to%20see%20the%20result.) and [here](https://docs.avohq.io/2.0/resources.html#:~:text=You%20can%20see%20the%20result%20in%20the%20admin%20panel%20using%20this%20URL%20/avo.%20The%20Post%20resource%20will%20be%20visible%20on%20the%20left%20sidebar.)

Thank you!

**Reply by brandonzylstra** [reply]
Thank you!

**Reply by adrianthedev** [reply]
Thank you for mentioning it. Keep them coming please!
